test/box-luatest/session_test.lua

```diff
@@ -0,0 +1,311 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_each(function(cg)
+    cg.server = server:new({
+        alias = 'session',
+        box_cfg = {
+            memtx_memory =  50 * 1024 * 1024,
+        },
+    })
+    cg.server:start()
+    cg.server:exec(function()
+        local space = box.schema.space.create('tweedledum')
+        space:create_index('primary', { type = 'hash' })
+    end)
+end)
+
+g.after_each(function(cg)
+    cg.server:drop()
+end)
+
+g.test_sanity_check_session = function(cg)
+    cg.server:exec(function()
+        local session = box.session
+        t.assert(session.exists(session.id()), 'session is created')
+        t.assert_equals(session.peer(session.id()), 'unix/:(socket)',
+                        'session.peer')
+        t.assert(session.exists(), 'session.exists')
+        local _, err = pcall(session.exists, 1, 2, 3)
+        t.assert_equals(tostring(err), 'session.exists(sid): bad arguments',
+                        'exists bad args #2')
+        t.assert(not session.exists(1234567890), 'session doesn\'t exist')
+    end)
+end
+
+g.test_check_session = function(cg)
```

Comment:
Minor: I would rather name it `g.test_check_session_id` instead.
Feel free to ignore.